src/forms/labels/labeled-field.js

```diff
@@ -7,9 +7,9 @@ import { hasInputError } from '../helpers'
 
 /**
  *
- * A fieldset wrapper for redux-form controlled inputs. This wrapper adds a label component (defaults to {@link InputLabel})
+ * A container wrapper for redux-form controlled inputs. This wrapper adds a label component (defaults to {@link InputLabel})
```

Comment:
IMO "container wrapper" is redundant. It's a wrapper or a container, but not both?